### PR TITLE
set the until parameter to the last second of the day when it's parsed

### PR DESCRIPTION
**Pull request by LBHmsaitz**
Fixed a bug that makes the date filtering not returning work orders from the current day. 
Now when the parameter 'until' is passed to the controller, the datetime result of the parsing is set to be at the very last second of the day

### CODE REVIEW

File: HackneyRepairs/Repository/UHWWarehouseRepository.cs [hunk 1]
```diff
@@ -70,7 +70,7 @@ public async Task<IEnumerable<RepairRequestBase>> GetRepairRequests(string prope
             catch (Exception ex)
             {
                 _logger.LogError(ex.Message);
-                throw new UhtRepositoryException();
+                throw new UhwRepositoryException();
             }
         }
```
Comment (zuren) [inline HackneyRepairs/Repository/UHWWarehouseRepository.cs]: Are these changes related?

Comment (LBHmsaitz) [reply HackneyRepairs/Repository/UHWWarehouseRepository.cs]: They're not, I found them and as they're one liners I've fixed them on the spot. Perhaps I should have commit them separately
